pkg/controllers/managementuser/alert/watcher/cluster_scan.go

```diff
@@ -50,7 +50,10 @@ func (csw *ClusterScanWatcher) Sync(_ string, cs *v3.ClusterScan) (runtime.Objec
 	if cs == nil {
 		return nil, nil
 	}
-	if cs.DeletionTimestamp != nil {
+	// Cis sends alerts when the cluster scan has completed, and when it has completed with failures.
+	// In both scenarios, the "CisScanStatus" needs to be set in order to send an alert, so we skip
+	// sending alerts until then.
+	if cs.DeletionTimestamp != nil || cs.Status.CisScanStatus == nil {
```

Comment:
nit: I'd probably just separate these checks since the comment only apply to one of them.

Reply:
Hmmmmmmmm, I concur, Ricky. Will do.

Reply:
Not the cleanest thing ever, but I think the readability is more important here, and agree with you.